site/public/js/simple-grading.js

```diff
@@ -4,17 +4,18 @@
 function updateVisibility() {
     const showGraders = $('#show-graders').is(':checked');
     const showDates = $('#show-dates').is(':checked');
-    $('.cell-grade').each(() => {
-        const graderElement = $(this).find('.simple-grade-grader');
-        const dateElement = $(this).find('.simple-grade-date');
-        if (showGraders && graderElement.text().trim() !== '') {
+    $('.cell-grade').each((idx, elem) => {
+        const graderElement = $(elem).find('.simple-grade-grader');
+        const dateElement = $(elem).find('.simple-grade-date');
+        const score = $(elem).attr('data-score');
+        if (showGraders && graderElement.text().trim() !== '' && score !== '0') {
```

Comment:
What if the grader changed a 2 to a 0. I would think it would be useful to see who gave them the 0. Due to the current backend design, the first grader on each row sets the rest of the row to 0s. I would argue that the issue isn't really an issue since it is correct that they received a 0 on everything else. However, if its something we want addressed, there would need to be some backend changes.